modules/mono/glue/GodotSharp/GodotSharp/Core/Aabb.cs

```diff
@@ -723,7 +723,7 @@ public Aabb(real_t x, real_t y, real_t z, real_t width, real_t height, real_t de
         /// <returns>A hash code for this AABB.</returns>
         public override readonly int GetHashCode()
         {
-            return _position.GetHashCode() ^ _size.GetHashCode();
+            return Hasher.Hash(_position, _size);
```

Comment:
```suggestion
            return HashCode.Combine(_position, _size);
```

Same as for all the others, use `System.HashCode.Combine` instead of creating a custom solution.